notebooks/official/workbench/pricing_optimization/pricing-optimization.ipynb

```diff
@@ -0,0 +1,2206 @@
+{
```

Comment:
<p>use notebook template</p><p>combine installs to single cell</p><p>add -q option to pip3</p><p> </p><p>remove execution output throughout notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>